content/ui-patterns/degraded-experiences.mdx

```diff
@@ -0,0 +1,335 @@
+      src="https://github.com/primer/design/assets/2313998/0e4db742-53cf-4671-bd9d-eca8ad316929"
+    />
+    <Caption>Don't hide buttons that part of core workflows.</Caption>
+  </Dont>
+</DoDontContainer>
+
+#### Indicating a button is non-functional without disabling it
+
+{/* <!-- TODO: Decide whether this should be added to the component API, or use the existing Button API. If so, maybe its it's own component that composes Button+Tooltip+Dialog. --> */}
+
+<Box
+  mb={3}
+  display="flex"
+  alignItems="flex-start"
+  justifyContent="space-between"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 3}}
+>
+
+<div>
+
+If a button is too critical to be omitted and responds to user input by showing more info about why it's non-functional, use an inert button. An inert button has the same styles as a disabled button, but it still reponds to user input.
+
+An inert button is the accessible option because buttons may not be disabled if they respond to user input. For example, if a button shows a tooltip when hovered or focused, or a dialog when clicked.
+
+</div>
+
+<img
+  width="456"
+  alt="A default button and an inert button"
+  src="https://github.com/primer/design/assets/2313998/fd1a7c99-b8c3-43fd-9f74-ae509e9efa31"
+/>
+
+</Box>
+
+There are two recommended ways an inert button should respond to user input:
+
+1. **Show a dialog on click:** When a user tries to activate an inert the button, open a [dialog](/components/dialog) that explains why the action cannot be completed and give them a path forward if possible. It is required to provide some kind of feedback when a user clicks a button.
+2. **Show a tooltip on hover or focus:** Before a user tries to activate a non-functional control, a [tooltip](/components/tooltip) with additional context may be displayed on hover or focus. It is _not_ required to respond to hover and focus.
+
+<img
+  width="960"
+  alt="A video of a 'broken' button that shows a tooltip and opens an error dialog when activated"
+  src="https://github.com/primer/design/assets/2313998/7786d3f9-c70c-4109-a49d-549231ccf79d"
+/>
+
+#### Disabling a non-functional button
+
+{/* <!-- TODO: Come up with a visual example where it's safe to disable the button. --> */}
+
+<img
+  width="960"
+  alt="Example of a button that can be disabled because it has sufficient context explaining why it is disabled"
+  src="https://github.com/primer/design/assets/2313998/e8ec073d-b0c9-4f41-aa6c-dae29aaf7fca"
+/>
+
+If a button is non-functional and there is already enough context for the user to understand why it's disabled, there's no need to respond to user input, so the button can safely be disabled.
+
+For more info, see the guidance on [disabling links and buttons](https://accessibility-playbook.github.com/link-and-button-guidance#disabling-links-and-buttons) from GitHub's Accessibility Playbook (only available to GitHub staff).
+
+### Handling action menus with non-functional items
+
+<img
```

Comment:
The spacing in the menus are too little. [You can find them in Figma here if that helps](https://www.figma.com/file/rnVzGIqsDvJ3ZwoNqJFdAn/ActionList-Menu-Documentation?type=design&node-id=63%3A19250&mode=design&t=3xyz6Xu8E8nWKJUC-1).

Reply:
It would be good to get more feedback on this because I find this a bit much. Maybe a dialog explaining those aren't available upon clicking an item would be better...

Reply:
> Maybe a dialog explaining those aren't available upon clicking an item would be better...

I considered that, but I was concerned that it might be frustrating to have to click and open a dialog to get more context.